### PR TITLE
[TW-1391] migration guide curl

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Added a new doc for importing cURL commands. Note that I added a leftnav item for a separate doc to avoid a merge conflict. That doc (Importing from Swagger) is in its own PR.

### CODE REVIEW

File: src/pages/docs/getting-started/importing-and-exporting/importing-curl-commands.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+cURL is an efficient command line tool for making HTTP requests. When testing a web application or API, cURL enables you to interact directly from the command line, using a well-established syntax that's common in the community. If you have several cURL commands stashed in different places, you can import them into Postman.
+
+## Contents
+
+* [Import a cURL command into Postman](#import-a-curl-command-into-postman)
+* [Convert a Postman request to a cURL command](#convert-a-postman-request-to-curl)
+
+## Import a cURL command into Postman
+
+1. Select **Import** in the left navigation menu. The **Import** modal opens.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-curl-commands.md:34]:
```suggestion
1. Select **Import** in the sidebar. The **Import** modal opens.
```

File: src/pages/docs/getting-started/importing-and-exporting/importing-curl-commands.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+1. Paste your cURL command into the input field at the top. Postman creates and opens a request automatically.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-curl-commands.md:35]:
```suggestion
1. Paste your cURL command into the box at the top. Postman creates and opens a request automatically.
```

File: src/pages/docs/getting-started/importing-and-exporting/importing-curl-commands.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+1. Open the request you want to use for a code snippet, then select the code icon <img alt="Code icon" src="https://assets.postman.com/postman-docs/icon-code-snippet.jpg#icon" width="16px"> in the right pane.
+
+    <img src="https://assets.postman.com/postman-docs/v10/generate-code-right-sidebar-v10.jpg" width="300px" alt="Open code snippets"/>
+
+1. Select **cURL** from the dropdown list.
+1. Select the copy icon <img alt="Copy icon" src="https://assets.postman.com/postman-docs/icon-copy-v9.jpg#icon" width="15px"> to copy the code snippet to your clipboard.
+
+    ![Copy code snippet to clipboard](https://assets.postman.com/postman-docs/v10/code-snippet-copy-icon-v10.jpg)
+
+    > If you’re a CLI based developer, and you prefer working inside the command line with cURL making basic requests, the [VS code extension](/docs/getting-started/basics/about-vs-code-extension/) is for you.
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-curl-commands.md:52]:
```suggestion
    > If you prefer working inside the command line with cURL making basic requests, the [VS code extension](/docs/getting-started/basics/about-vs-code-extension/) is for you.
```

**Review by jkonrath-postman — APPROVED** [review]
A couple of little things, but LGTM

File: src/pages/docs/getting-started/importing-and-exporting/importing-curl-commands.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+1. Select **Import** in the left navigation menu. The **Import** modal opens.
+1. Paste your cURL command into the input field at the top. Postman creates and opens a request automatically.
+
+    > You can also paste cURL commands into a request's URL field.
+
+## Convert a Postman request to cURL
+
+To convert a Postman request to a cURL command, do the following:
+
+1. Open the request you want to use for a code snippet, then select the code icon <img alt="Code icon" src="https://assets.postman.com/postman-docs/icon-code-snippet.jpg#icon" width="16px"> in the right pane.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-curl-commands.md:43]:
```suggestion
1. Open the request you want to use for a code snippet, then select the code icon <img alt="Code icon" src="https://assets.postman.com/postman-docs/icon-code-snippet.jpg#icon" width="16px"> in the right sidebar.
```

**Review by mark-anthony-postman — APPROVED** [review]
Great new page! 🌟 Noted a couple UI terms to update

**Pull request merged**